src/cloud-api-adaptor/podvm-mkosi/mkosi.skeleton/usr/lib/repart.d/30-scratch.conf

```diff
@@ -0,0 +1,5 @@
+[Partition]
+Type=linux-generic
+Label=scratch
+Encrypt=key-file
+Format=ext4
```

Comment:
@mkulke can you please explain how the size of the partition gets calculated ? Is it automatically done by systemd?

Reply:
by default, it grabs as much as it can get on the disk.

if the size of the disk is 1GB it's about 0.5gb
if the size of the disk is 10gb it's about 9.5gb